blockchain/blockchain_test.go

```diff
@@ -966,7 +986,8 @@ func TestReorgSideEvent(t *testing.T) {
 
 	chain, _ := GenerateChain(gspec.Config, genesis, faker.NewFaker(), db, 3, func(i int, gen *BlockGen) {})
 	if _, err := blockchain.InsertChain(chain); err != nil {
-		t.Fatalf("failed to insert chain: %v", err)
+		errs = append(errs, fmt.Errorf("failed to insert chain: %v", err))
```

Comment:
Try using [errors.Join](https://pkg.go.dev/errors#Join)

Reply:
In this thread, we conclude a err is enough. Thank you for your suggestion.
https://github.com/kaiachain/kaia/pull/690#discussion_r2631437488